homeassistant/helpers/data_entry_flow.py

```diff
@@ -18,7 +18,7 @@
 class _BaseFlowManagerView(HomeAssistantView):
     """Foundation for flow manager views."""
 
-    def __init__(self, flow_mgr: data_entry_flow.BaseFlowManager) -> None:
+    def __init__(self, flow_mgr: data_entry_flow.FlowManager) -> None:
```

Comment:
I'm not sure the typing is correct now, do we need to specify this is not a `FlowManager` specialized to the default FlowResult?

Reply:
The best option is to make `_BaseFlowManagerView` itself generic for `FlowManager`. Quite a lot of generics :)
Though it might be easier if I pushed a commit to do that: https://github.com/home-assistant/core/pull/112345/commits/d27640a16a5a22428dee9ef5b54c26a25e80a1c7